### PR TITLE
Edit Date 2016 > 2018 ! 

### PR DESCRIPTION
**Pull request by sctnightcore**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [ ] I have followed [proper Hercules code styling][code].
- [ ] I have read and understood the [contribution guidelines][cont] before making this PR.
- [ ] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 

[//]: # (Master? Slave?)

**Issues addressed:**

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2053)

File: src/char/int_rodex.c [hunk 1]
```diff
@@ -216,7 +216,7 @@ static int inter_rodex_fromsql(int char_id, int account_id, int8 opentype, int64
 #else
 			msg.opentype = opentype;
 #endif
-#if PACKETVER < 20160601
+#if PACKETVER < 20180601
```
Comment (Emistry) [inline src/char/int_rodex.c]: I believe this is not an intended changes.
try not to search and replace using the sole keyword `2016`.

File: src/map/itemdb.c [hunk 2]
```diff
@@ -2079,7 +2079,7 @@ int itemdb_readdb_libconfig_sub(struct config_setting_t *it, int n, const char *
 	if (itemdb->lookup_const(it, "ViewSprite", &i32) && i32 >= 0)
 		id.view_sprite = i32;
 
-	if (itemdb->lookup_const(it, "View", &i32) && i32 >= 0) { // TODO: Remove (Deprecated - 2016-09-04 [Haru])
+	if (itemdb->lookup_const(it, "View", &i32) && i32 >= 0) { // TODO: Remove (Deprecated - 2018-09-04 [Haru])
```
Comment (Emistry) [inline src/map/itemdb.c]: I believe this is not an intended changes.
try not to search and replace using the sole keyword `2016`.

File: src/char/int_rodex.c (hunk 1 above)
Comment (sctnightcore) [reply src/char/int_rodex.c]: wait pls

File: src/map/itemdb.c (hunk 2 above)
Comment (sctnightcore) [reply src/map/itemdb.c]: wait pls

File: src/char/int_rodex.c (hunk 1 above)
Comment (sctnightcore) [reply src/char/int_rodex.c]: Fixed : 5369c79

File: src/map/itemdb.c (hunk 2 above)
Comment (sctnightcore) [reply src/map/itemdb.c]: Fixed : 5369c79

**Comment by 4144** [comment]
pr must be without revert commits
this mean squash all your changes into one commit

**Comment by Emistry** [comment]
no nid to close... just squash the commits into one.

**Comment by sctnightcore** [comment]
I like to start a new one.

**Comment by Emistry** [comment]
it make the closed PR list messy and redundant tho, and extra works for you to recreate the branch and apply changes.

**Pull request closed without merging**